lib/rules/i18n-no-reference.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const { I18N_METHODS } = require('../i18n.js');
+
+const isReferenceNode = (node) => {
+  if (node.type === 'TemplateLiteral') {
+    return false;
+  }
+
+  return node.type !== 'Literal' || typeof node.value !== 'string';
+};
+
+const extractI18nMethodAndArgs = (node) => {
+  let result = [];
+
+  if (node.type === 'CallExpression') {
+    const { callee } = node;
+    if (callee.type === 'MemberExpression') {
+      const { object, property } = callee;
+      if (object.name === 'i18n') {
+        result = [property.name, node.arguments];
+      }
+    }
+  }
+
+  return result;
+};
+
+module.exports = {
+  meta: {
+    docs: {
+      githubIssue: 'https://github.com/mozilla/eslint-plugin-amo/issues/232',
+      category: 'Possible Errors',
+      description:
+        'Ensure predictable static values are passed as i18n method arguments',
+      recommended: true,
+    },
+    fixable: null, // Indicates that some problems reported by this rule are automatically fixable
```

Comment:
I don't think this comment is accurate. WDYT?

Reply:
Fixed.